src/uct/base/uct_iface.h

```diff
@@ -530,5 +530,15 @@ size_t uct_iov_total_length(const uct_iov_t *iov, size_t iovcnt)
     return total_length;
 }
 
+/**
+ * Used to call uct_iov_total_length() in a non-release configuration
+ */
+#if (ENABLE_STATS || ENABLE_PARAMS_CHECK)
+#define UCT_IOV_TOTAL_LENGTH_CHECK(_total_length, _iov, _iovcnt) \
+    size_t _total_length = uct_iov_total_length(_iov, _iovcnt);
+#else
+#define UCT_IOV_TOTAL_LENGTH_CHECK(_total_length, _iov, _iovcnt)
```

Comment:
@yosefe I plan to use this macro in other PRs to address "length check in debug only" issue.